sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_content_safety/_content_safety.py

```diff
@@ -70,7 +70,7 @@ class ContentSafetyEvaluator(EvaluatorBase):
 
     def __init__(self, credential, azure_ai_project, eval_last_turn: bool = False, **kwargs):
         super().__init__(eval_last_turn=eval_last_turn)
-        self._parallel = kwargs.pop("parallel", True)
+        self._parallel = kwargs.pop("parallel", False)
```

Comment:
What's reason for this change?

Reply:
I need to make a bug for it, but the content safety eval experiences a bug similar to what I fixed in the `evaluate` function when it received optional inputs. This is due to that bug in PF's thread executor code. I need to add a similar workaround here.

Reply:
Added a comment noting the bug ticket number.